test/t_web/channels/feed_channel_test.exs

```diff
@@ -125,29 +125,9 @@ defmodule TWeb.FeedChannelTest do
              }
     end
 
-    test "with news", %{socket: socket, me: me} do
-      import Ecto.Query
-
-      {1, _} =
-        News.SeenNews
-        |> where(user_id: ^me.id)
-        |> Repo.delete_all()
-
-      assert {:ok, %{"news" => news}, socket} = join(socket, "feed:" <> me.id)
-
-      assert [_first_news_item = %{id: 2, story: story}] = news
-
-      assert [p] = story
-
-      assert %{"background" => %{"color" => _}, "labels" => _, "size" => _} = p
-
-      ref = push(socket, "seen", %{"news_story_id" => 2})
-      assert_reply ref, :ok, _
-
+    test "with no news", %{socket: socket, me: me} do
```

Comment:
@ruslandoga do you think:
a) user with older inserted_at should be factory-created here 
b) news should be tested separately as a module
c) sth else?

Reply:
`%User{user | id: old_id}`